_rules/meta-refresh-no-delay-bc659a.md

```diff
@@ -84,8 +95,8 @@ First valid `<meta http-equiv="refresh">` redirects immediately.
 
 ```html
 <head>
-	<meta http-equiv="refresh" content="0; http://example.com" />
-	<meta http-equiv="refresh" content="5; http://example.com" />
+	<meta http-equiv="refresh" content="0; https://w3c.org" />
+	<meta http-equiv="refresh" content="5; https://w3c.org" />
```

Comment:
Don't think we need to change the examples. It invalidates implementation data for no real reason.

Reply:
The important change here is to use HTTPS rather than HTTP.

It was requested in https://github.com/act-rules/act-rules.github.io/pull/1527#discussion_r555222476 and https://github.com/act-rules/act-rules.github.io/pull/1527#discussion_r555222592

Reply:
Alright, fair enough.